plugin-dell/dphandler/storage_test.go

```diff
@@ -43,6 +43,17 @@ func mockDevice(username, password, url string, w http.ResponseWriter) {
 		Name:         "Volume Collection",
 	}
 
+	firmware := dpmodel.FirmwareVersion{
+		FirmwareVersion: "4.40.10.00",
+	}
```

Comment:
Please add a case for negative scenario too

Reply:
Added